From ae4adbccb0936c70052d1993e1f7837b567e56f9 Mon Sep 17 00:00:00 2001
From: Alan Pevec <apevec@redhat.com>
Date: Fri, 4 Oct 2013 17:04:00 +0200
Subject: [PATCH] rename httpd/keystone.py to httpd/keystone.wsgi

This matches openstack-dashboard[1]
.py is misleading since this isn't usable outside mod_wsgi

[1] https://github.com/openstack/horizon/tree/master/openstack_dashboard/wsgi

Change-Id: I1ba7f0e0f72d5c9fa1d06cec2b50551eb7f2e296
(cherry picked from commit b22932feaf137b24475e10efff64d899db3573e0)
---
 doc/source/apache-httpd.rst |  4 ++--
 httpd/keystone.py           | 52 ---------------------------------------------
 httpd/keystone.wsgi         | 52 +++++++++++++++++++++++++++++++++++++++++++++
 3 files changed, 54 insertions(+), 54 deletions(-)
 delete mode 100644 httpd/keystone.py
 create mode 100644 httpd/keystone.wsgi

diff --git a/doc/source/apache-httpd.rst b/doc/source/apache-httpd.rst
index f4b6cb5..7821f66 100644
--- a/doc/source/apache-httpd.rst
+++ b/doc/source/apache-httpd.rst
@@ -67,9 +67,9 @@ Copy the file httpd/wsgi-keystone.conf to the appropriate location for your apac
 
     /etc/httpd/conf.d/wsgi-keystone.conf
 
-Create the directory ``/var/www/cgi-bin/keystone/``. You can either hardlink or softlink the files ``main`` and ``admin`` to the file ``keystone.py`` in this directory.  For a distribution appropriate place, it should probably be copied to::
+Create the directory ``/var/www/cgi-bin/keystone/``. You can either hardlink or softlink the files ``main`` and ``admin`` to the file ``keystone.wsgi`` in this directory.  For a distribution appropriate place, it should probably be copied to::
 
-    /usr/share/openstack/keystone/httpd/keystone.py
+    /usr/share/keystone/keystone.wsgi
 
 Keystone's primary configuration file (``etc/keystone.conf``) and the PasteDeploy
 configuration file (``etc/keystone-paste.ini``) must be readable to HTTPD in
diff --git a/httpd/keystone.py b/httpd/keystone.py
deleted file mode 100644
index 50ae07e..0000000
--- a/httpd/keystone.py
+++ /dev/null
@@ -1,52 +0,0 @@
-# vim: tabstop=4 shiftwidth=4 softtabstop=4
-
-# Copyright 2013 OpenStack Foundation
-#
-#    Licensed under the Apache License, Version 2.0 (the "License"); you may
-#    not use this file except in compliance with the License. You may obtain
-#    a copy of the License at
-#
-#         http://www.apache.org/licenses/LICENSE-2.0
-#
-#    Unless required by applicable law or agreed to in writing, software
-#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
-#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
-#    License for the specific language governing permissions and limitations
-#    under the License.
-
-import logging
-import os
-
-from paste import deploy
-
-from keystone.openstack.common import gettextutils
-
-# NOTE(blk-u):
-# gettextutils.install() must run to set _ before importing any modules that
-# contain static translated strings.
-#
-# Configure gettextutils for deferred translation of messages
-# so that error messages in responses can be translated according to the
-# Accept-Language in the request rather than the Keystone server locale.
-gettextutils.install('keystone', lazy=True)
-
-from keystone.common import environment
-from keystone import config
-from keystone.openstack.common import log
-
-
-CONF = config.CONF
-CONF(project='keystone')
-config.setup_logging(CONF)
-
-environment.use_stdlib()
-name = os.path.basename(__file__)
-
-if CONF.debug:
-    CONF.log_opt_values(log.getLogger(CONF.prog), logging.DEBUG)
-
-# NOTE(ldbragst): 'application' is required in this context by WSGI spec.
-# The following is a reference to Python Paste Deploy documentation
-# http://pythonpaste.org/deploy/
-application = deploy.loadapp('config:%s' % config.find_paste_config(),
-                             name=name)
diff --git a/httpd/keystone.wsgi b/httpd/keystone.wsgi
new file mode 100644
index 0000000..50ae07e
--- /dev/null
+++ b/httpd/keystone.wsgi
@@ -0,0 +1,52 @@
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+
+# Copyright 2013 OpenStack Foundation
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import logging
+import os
+
+from paste import deploy
+
+from keystone.openstack.common import gettextutils
+
+# NOTE(blk-u):
+# gettextutils.install() must run to set _ before importing any modules that
+# contain static translated strings.
+#
+# Configure gettextutils for deferred translation of messages
+# so that error messages in responses can be translated according to the
+# Accept-Language in the request rather than the Keystone server locale.
+gettextutils.install('keystone', lazy=True)
+
+from keystone.common import environment
+from keystone import config
+from keystone.openstack.common import log
+
+
+CONF = config.CONF
+CONF(project='keystone')
+config.setup_logging(CONF)
+
+environment.use_stdlib()
+name = os.path.basename(__file__)
+
+if CONF.debug:
+    CONF.log_opt_values(log.getLogger(CONF.prog), logging.DEBUG)
+
+# NOTE(ldbragst): 'application' is required in this context by WSGI spec.
+# The following is a reference to Python Paste Deploy documentation
+# http://pythonpaste.org/deploy/
+application = deploy.loadapp('config:%s' % config.find_paste_config(),
+                             name=name)
